odl/test/discr/discr_ops_test.py

```diff
@@ -242,5 +235,27 @@ def test_resizing_op_adjoint(padding, fn_impl):
         assert almost_equal(inner1, inner2, places=dtype_places(dtype))
 
 
+def test_resizing_op_mixed_uni_nonuni():
+    """Check if resizing along uniform axes in mixed discretizations works."""
+    nonuni_part = odl.nonuniform_partition([0, 1, 4])
+    uni_part = odl.uniform_partition(-1, 1, 4)
+    part = uni_part.append(nonuni_part).append(uni_part).append(nonuni_part)
+    fspace = odl.FunctionSpace(odl.IntervalProd(part.min_pt, part.max_pt))
+    dspace = odl.rn(part.size)
+    space = odl.DiscreteLp(fspace, part, dspace)
+
+    # Keep non-uniform axes fixed
+    res_op = odl.ResizingOperator(space, ran_shp=(6, 3, 6, 3))
+
+    assert res_op.axes == (0, 2)
+    assert res_op.offset == (1, 0, 1, 0)
+
```

Comment:
No evaluation test at all? Only testing the adjoint is scary since this may as well be the zero operator and we'll never know.